commands/command_e2e_test.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+)
+
+func (s *MmctlE2ETestSuite) TestMoveCommandCmdF() {
+	s.SetupTestHelper().InitBasic()
+
+	// create new command
+	newCmd := &model.Command{
+		CreatorId: s.th.BasicUser.Id,
+		TeamId:    s.th.BasicTeam.Id,
+		URL:       "http://nowhere.com",
+		Method:    model.COMMAND_METHOD_POST,
+		Trigger:   "trigger",
+	}
+	command, _ := s.th.SystemAdminClient.CreateCommand(newCmd)
+
+	s.RunForAllClients("move command to non existing team", func(c client.Client) {
+		printer.Clean()
+
+		team := "nonexisting team"
+		err := moveCommandCmdF(s.th.SystemAdminClient,
+			&cobra.Command{},
+			[]string{team, command.Id})
+		s.Require().NotNil(err)
+		s.Require().Equal(fmt.Sprintf("unable to find team '%s'", team), err.Error())
+	})
+
+	s.RunForAllClients("move non existing command", func(c client.Client) {
+		printer.Clean()
+
+		err := moveCommandCmdF(s.th.SystemAdminClient,
```

Comment:
Same as in the previous test case and comment from @isacikgoz, this should be:

```suggestion
		err := moveCommandCmdF(c,
```